tests/draft2019-09/optional/refOfUnknownKeyword.json

```diff
@@ -2,7 +2,7 @@
     {
         "description": "reference of a root arbitrary keyword ",
         "schema": {
-            "$schema": "https://json-schema.org/draft/2019-09/schema",
+            "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
..and the same here, they should be draft2019-09.

Reply:
Resolved. Wish I'd found this before I had to merge up the fork :-)